styles.css

```diff
@@ -1,3 +1,105 @@
+:root {
+    --dark-purple: #1F1F3C;
+    --medium-purple: #A8AAD2;
+    --light-purple: #EAEAF4;
+    --white: #FFFFFF;
+}
+
+* {
+    margin: 0;
+}
+
 body {
     font-family: 'Poppins', sans-serif;
+    display: flex;
+    height: 100vh;
+}
+
+nav {
+    background-color: var(--dark-purple);
+    width: 30vw;
+    display: flex;
+    justify-content: center;
+}
+
+h1 {
+    color: var(--light-purple);
+    font-size: 4.2em;
+    font-weight: bold;
+    margin-top: 14px;
+}
+
+main {
+    width: 100%;
+    display:flex;
+    flex-direction: column;
+}
+
+form {
+    width: 100%;
+    display: flex;
+    align-content: flex-end;
+}
+
+.topPane, form {
+    height: 50vh;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: 100%;
+    background-color: var(--medium-purple);
+    align-content: flex-end; 
+    font-size: 1.3em;  
+}
+
+.inputContainer{
+    display:flex;
+    justify-content: flex-end;
+    padding-right: 6em;
+}
+
+input,textarea,button {
+    width: 53vw;
+}
+
+label {
```

Comment:
There is a way to center the labels to the left side of the inputs they correspond with. This is a resource I've used in the past to accomplish this. https://www.w3docs.com/snippets/html/how-to-align-labels-next-to-inputs.html